pkg/utils/spi/controller.go

```diff
@@ -95,3 +110,81 @@ func (s *SuiteController) GetSPIAccessToken(name, namespace string) (*spi.SPIAcc
 	}
 	return &spiAccessToken, nil
 }
+
+// Inject manually access tokens using spi API
+func (s *SuiteController) InjectManualSPIToken(namespace string, repoUrl string, oauthCredentials string, secretType v1.SecretType) string {
+	var spiAccessTokenBinding *v1beta1.SPIAccessTokenBinding
+	var accessTokenSecret = util.GenerateRandomString(10)
+
+	// Get the token for the current openshift user
+	bearerToken, err := utils.GetOpenshiftToken()
+	Expect(err).NotTo(HaveOccurred())
+
+	spiAccessTokenBindingName, err := s.CreateSPIAccessTokenBinding(SPIAccessTokenBindingPrefixName, namespace, repoUrl, accessTokenSecret, secretType)
```

Comment:
The naming is weird. This variable is SPIAccessTokenBinding instance, not its name. 
Maybe we don't need two variables for SPIAccessTokenBinding  and we can merge `spiAccessTokenBindingName` and `spiAccessTokenBinding`? I've quickly tried that and didn't encounter any problems.